src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -1275,7 +1270,7 @@ void FancyZones::RegisterVirtualDesktopUpdates(std::vector<GUID>& ids) noexcept
     std::unique_lock writeLock(m_lock);
 
     m_workAreaHandler.RegisterUpdates(ids);
-    std::vector<std::wstring> active{};
+    std::vector<GUID> active{};

```

Comment:
Argument `std::vector<GUID>& ids` is what we should use here. Now that we no longer use strings, this additional call to `VirtualDesktopUtils::GetVirtualDesktopIds` is obsolete.

Reply:
👍 done